frontend/apps/webapp/src/core/components/Branding/BrandingProvider.tsx

```diff
@@ -1,9 +1,9 @@
-import axios, { AxiosResponse } from 'axios';
+// import axios, { AxiosResponse } from 'axios';
```

Comment:
Let's delete, instead of commenting out the line.  
If we need to, we can view the history of the file to see the old code.